wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioEffectsPickerViewController.swift

```diff
@@ -188,13 +188,14 @@ final class AudioEffectsPickerViewController: UIViewController {
 
     private func loadLevels() {
         let url = URL(fileURLWithPath: recordingPath)
-        fileMetadataGenerator.metadataForFileAtURL(url, UTI: url.UTI(), name: url.lastPathComponent) { metadata in
-            DispatchQueue.main.async(execute: {
+        Task {
+            let metadata = await fileMetadataGenerator.metadataForFile(at: url)
+            await MainActor.run {
```

Comment:
Could we do `Task { @MainActor  in {...} } here to be aligned with what you did right after (line 83 in ConversationInputBarViewController+Files) ?